### ISSUE TITLE
Cannot reproduce your work

### ISSUE THREAD

**Issue by CloudRR** [mlfoundations/open_clip#300]
Hi team,
I am trying to reproduce the numbers of your model on the Conceptual Captions dataset and am not able to reproduce the same numbers. 

For example, I obtain a 0.0532 imagenet zeros shot top 1 val accuracy with RN50 config compares to the ~0.2 you report in the chapter of Loss Curves in README as follows. 
![image](https://user-images.githubusercontent.com/31526283/208014383-077bb75f-a0be-405c-b50b-1e71fbd6509a.png)

I used dataset of cc12m **but only about 3M data pair** is got due to limited network. I used one node of 8 V100 GPUs with a batchsize of  150*8=1200 with default params(32 epoch)

My command is :
torchrun --nproc_per_node 8 --nnodes=1 --node_rank=0 -m training.main --train-data '/dataset/cc12m/{00000..00640}.tar' --dataset-type webdataset --batch-size 150 --precision amp --workers 4 --imagenet-val=/dataset/imagenet/val --model RN50 --train-num-samples 3056141 --report-to tensorboard

Could you please help me to find what's wrong?  Maybe it is the small dataset(3M data, 25% of cc12m)? or other  hyperparams(like lr)?

Could you share your other hyperparams you used to obtain those results so that I can run the training with the exact same setup, or better yet share the exact command you used to run the training runs specific to the reported results, I would really appreciate that.

Thanks a lot.

**Comment by rwightman** [comment]
@CloudRR yeah, something is wrong there but really hard to say what it is. BTW that 20% graph is also low, is that actually the one in the README for cc12m + RN50? RN50 should be ~ 36% top-1 zero-shot in-1k on cc12m. I also trained a convnext tiny a couple weeks ago and it rounds to 36% (similar flops to RN50).

Are you sure the images and text downlaoded are okay? Did you check, resolution, etc. with only 3M samples you'll definitely be less, I'd expect maybe 20%?

**Comment by rom1504** [comment]
> I used dataset of cc12m but only about 3M data pair is got due to limited network

I would definitely fix that first

**Comment by CloudRR** [comment]
> @CloudRR yeah, something is wrong there but really hard to say what it is. BTW that 20% graph is also low, is that actually the one in the README for cc12m + RN50? RN50 should be ~ 36% top-1 zero-shot in-1k on cc12m. I also trained a convnext tiny a couple weeks ago and it rounds to 36% (similar flops to RN50).
> 
> Are you sure the images and text downlaoded are okay? Did you check, resolution, etc. with only 3M samples you'll definitely be less, I'd expect maybe 20%?

Thanks a lot! I rechecked my dataset and  found a mistake of dataset path. I retrained with 3M samples from cc12m and got:

imagenet-zeroshot-val-top1: 0.2141  imagenet-zeroshot-val-top5: 0.4466, 

which is close to your prediction of 20%.  Is that normal?

**Comment by CloudRR** [comment]
> 

Thanks a lot!  I rechecked my dataset and found a mistake of dataset path. And it is the reason of dataset.
I am in China and in a limited company network.  I am trying to download cc12m using a proxy of HK now, hope more data will be got.

**Comment by CloudRR** [comment]
> @CloudRR yeah, something is wrong there but really hard to say what it is. BTW that 20% graph is also low, is that actually the one in the README for cc12m + RN50? RN50 should be ~ 36% top-1 zero-shot in-1k on cc12m. I also trained a convnext tiny a couple weeks ago and it rounds to 36% (similar flops to RN50).
> 
> Are you sure the images and text downlaoded are okay? Did you check, resolution, etc. with only 3M samples you'll definitely be less, I'd expect maybe 20%?

Is this picture of your Readme actually trained with cc3m? It got about 20% top-1 zero-shot, which is very similar to what I have got with about 3M data

**Comment by gabrielilharco** [comment]
Yes, I believe that plot is with CC3M, not CC12M. @carlini might remember better. If it's indeed CC3M, sounds like the results you're getting are close to expected

**Comment by carlini** [comment]
That picture in the README was CC3M I believe yes.

**Comment by CloudRR** [comment]
> Yes, I believe that plot is with CC3M, not CC12M. @carlini might remember better. If it's indeed CC3M, sounds like the results you're getting are close to expected

Thanks a lot for your reply

**Comment by CloudRR** [comment]
> That picture in the README was CC3M I believe yes.

Thanks a lot for your reply
